internal/indexnode/indexnode.go

```diff
@@ -90,6 +102,15 @@ func (i *IndexNode) Register() error {
 	return nil
 }
 
+func (i *IndexNode) initKnowhere() {
```

Comment:
indexnode should not call SegcoreInit, instead you should add IndexBuilderInit and call it.

Reply:
OK, get it.